cmd/tailscale/cli/configure-kube.go

```diff
@@ -39,6 +41,19 @@ The hostname argument should be set to the Tailscale hostname of the peer runnin
 	Exec: runConfigureKubeconfig,
 }
 
+// kubeconfigBaseDir returns the base directory where the .kube directory
+// should exist.
+func kubeconfigBaseDir() string {
+	if runtime.GOOS == "windows" {
+		homedir.HomeDir()
+	}
+	h := os.Getenv("HOME")
+	if version.IsSandboxedMacOS() {
+		h, _, _ = strings.Cut(h, "/Library")
```

Comment:
Maybe add a comment with the expected value and explain that we depend on the entitlement to be able to read outside of the container?

May also want to cut `/Library/Containers` in case someone names their local user `Library` 🙃

Reply:
Done